bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/AbstractMyStromHandler.java

```diff
@@ -73,15 +62,14 @@ public abstract class AbstractMyStromHandler extends BaseThingHandler {
 
     public AbstractMyStromHandler(Thing thing, HttpClient httpClient) {
         super(thing);
+        config = getConfigAs(MyStromConfiguration.class);
         this.httpClient = httpClient;
     }
 
     @Override
     public final void initialize() {
-        MyStromConfiguration config = getConfigAs(MyStromConfiguration.class);
-        this.hostname = HTTP_REQUEST_URL_PREFIX + config.hostname;
-
         updateStatus(ThingStatus.UNKNOWN);
+        config = getConfigAs(MyStromConfiguration.class);
```

Comment:
This could be moved ahead of `updateStatus()` like before, since it's part of the initialization. `initialize()` needs to update status before leaving, and the last moment to do that is before scheduling the async task.